public/scss/components/PopUpVoucherCoupon.module.scss

```
@@ -159,6 +133,70 @@
   margin-top: 24px !important;
 }
 
+.voucherContainer
+{
+  @include fixedWidth(100%);
+  @include fixedHeight(100vh);
+  position: relative;
+  margin: 0;
+  padding: 16px;
+  background-color: $color_white;
+  border-radius: 2px;
+  overflow-y: auto;
+
+  @media screen and (min-width: #{$breakpoint_min_md})
+  {
+    padding: 24px;
+  }
+
+  @media screen and (min-width: #{$breakpoint_min_lg})
+  {
+    padding: 24px 135px;
+  }
+}
+
+.voucherTitleInput
+{
+  @include typography_builder(500, 16, 24);
+  margin: 0;
+}
+
+.voucherSubTitleInput
+{
+  @include typography_builder(400, 10, 16);
+  margin: 0;
+}
+
+.voucherInputContainer
+{
+  @include flex(row, center, flex-start);
+  height: 44px;
+  margin-top: 12px;
+
+  input[class*="form-control"]
+  {
+    @include transition();
+    @include typographyBuilder(400, 16, 16);
+    width: 231px;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
```

```suggestion
    @include fixedHeight(100%);
```

udah ada mixin ini kan ya kalo urban